src/AnimalSoundsApp.java

```diff
@@ -1,2 +1,56 @@
+// Definim interfața Animal
+interface Animal {
+    String getType();
+    String getSound();
+}
+
+// Implementare pentru un câine
+class Dog implements Animal {
+    @Override
+    public String getType() {
+        return "Dog";
+    }
+
+    @Override
+    public String getSound() {
+        return "Woof";
+    }
+}
+
+// Implementare pentru o pisică
```

Comment:
clasa de numeste Cat - stiu ca este o pisica; daca vin si modific numele clasei in Crocodile si las comment ul tau, pentru ca ma grabesc, ce poate intelege un om care se uita peste cod in 5 luni?